fmp_crawler/insider_crawler.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+from datetime import datetime, timedelta
+
+from base_fmp_crawler import BaseFMPCrawler
+import time
+import asyncio
+import logging
+from tqdm import tqdm
+
+INSIDER_TRADING_MAX_LIMIT = 1000
+INSIDER_TRADING_MIN_PAGE = 0
+INSIDER_TRADING_MAX_PAGE = 100
+
+
+class InsiderCrawler(BaseFMPCrawler):
```

Comment:
NIT: InsiderTradingCrawler is another good name if you prefer, so is for file name. : )